integration_tests/test_replay_block.py

```diff
@@ -92,3 +110,20 @@ def test_replay_block(custom_cronos):
     replay_receipts = [AttributeDict(receipt_formatter(item)) for item in rsp["result"]]
     assert replay_receipts[1].status == 0
     assert replay_receipts[1].gasUsed == gas_limit
+
+    # patch the unlucky tx with the new cli command
+    # stop the node0
+    end_height = cli.block_height()
+    supervisorctl(custom_cronos.base_dir / "../tasks.ini", "stop", "cronos_777-1-node0")
+    cli = custom_cronos.cosmos_cli()
+    results = cli.fix_unlucky_tx(begin_height, end_height)
+    # the second tx is patched
+    assert results[0][1] == txhashes[1].hex()
+    # start the node0 again
+    supervisorctl(
+        custom_cronos.base_dir / "../tasks.ini", "start", "cronos_777-1-node0"
+    )
+    # wait for tm-rpc port
+    wait_for_port(ports.rpc_port(custom_cronos.base_port(0)))
+    # check the tx indexer
+    assert len(cli.txs(f"ethereum_tx.ethereumTxHash={txhashes[1].hex()}")["txs"]) == 1
```

Comment:
This integration test only works before the root cause is fixed: https://github.com/crypto-org-chain/cronos/pull/503